### ISSUE TITLE
Maximum call stack size exceeded on pubsub Events

### ISSUE THREAD

**Issue by timestep** [balderdashy/sails#2954]
Hello all,

I would like to add that I'm encountering an identical issue to:
https://github.com/balderdashy/sails/issues/2636

Also possibly related are
https://github.com/balderdashy/sails/issues/2813
https://github.com/balderdashy/waterline/issues/891

```
/Users/kevin/Documents/dayPack/node_modules/sails/node_modules/lodash/dist/lodash.js:2415
    function isObject(value) {
                     ^
RangeError: Maximum call stack size exceeded
```

This occurs immediately after any attempt to use any pubsub method on the server or .message() event.

According to this commit,https://github.com/balderdashy/sails/commit/e4f2a2635562b6c96d7fc48b5dc42c3051b2a00e ,it is supposed to be fixed...

Sails v0.11.0
"sails": "0.11.0",
"socket.io-redis": "^0.1.4",

I can provide additional information if needed.
Any suggestions or possible solutions or oversights I have made please let me know. 

I am opening this ticket since there are not any particularly clear solutions to this issue nor issues that doesnt involve using _.clone or _.cloneDeep and seems to have been fixed in the master branch... However I just updated my sails version recently to v0.11.0.

**Comment by sgress454** [comment]
Fixed by https://github.com/balderdashy/sails/pull/2865?

**Comment by Oldwo1f** [comment]
+1

**Comment by scott-wyatt** [comment]
Seems this is till an issue with a fresh sails install up to 0.11.1, perhaps an npm patch is needed?

**Comment by gunish** [comment]
Any workarounds for this ?

**Comment by sailsbot** [comment]
Thanks for posting, @timestep.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
